app/templates/components/event-table.hbs

```diff
@@ -0,0 +1,68 @@
+  <div class="row">
+    <div class="sixteen wide column">
+      <table class="ui tablet stackable celled table">
+        <thead>
+          <tr>
+            <th>Name</th>
+            <th>Date</th>
+            <th>Roles</th>
+            <th>Sessions</th>
+            <th>Speakers</th>
+            <th>Tickets</th>
+            <th>Public Url</th>
+            <th>Actions</th>
+          </tr>
+        </thead>
+        <tbody>
+          <tr>
+            <td>
+              <div><img src="http://placehold.it/150x150"></div>
+              <div>Sample Event</div>
+            </td>
+            <td>
+              <div>27-06-2017</div>
+              <div class="center aligned">to</div>
+              <div>30-06-2017</div>
+            </td>
+            <td>
+              <div class="ui ordered list">
+                <div class="item">sample@gmail.com (Organizer)</div>
+                <div class="item">sample2@gmail.com (Manager)</div>
+              </div>
+            </td>
+            <td>
+              <div class="ui list">
+                <div class="item">Drafts: 0</div>
+                <div class="item">Submitted: 0</div>
+                <div class="item">Accepted: 0</div>
+                <div class="item">Confirmed: 0</div>
+                <div class="item">Pending: 0</div>
+                <div class="item">Rejected: 0</div>
+              </div>
+            </td>
+            <td class = "center aligned">
+              2
+            </td>
+            <td>
+              <div class="ui bulleted list">
+                <div class="item">Premium (12/100)</div>
+                <div class="item">VIP (10/15)</div>
+                <div class="item">Normal (100/200)</div>
+                <div class="item">Free (100/500)</div>
+              </div>
+            </td>
+            <td>
+              localhost/event
+            </td>
+            <td>
+              <div class="ui vertical compact basic  buttons">
+                <button class="ui button" data-content="Edit event details"><i class="edit icon"></i></button>
```

Comment:
As you can see yourself, the icon is mis-aligned. You need to add `ui icon button` class to the button. And also, add the attribute `type="button"`